src/test/java/com/iota/iri/service/tipselection/impl/EntryPointSelectorImplTest.java

```diff
@@ -1,63 +1,82 @@
 package com.iota.iri.service.tipselection.impl;
 
 import com.iota.iri.Milestone;
+import com.iota.iri.Snapshot;
+import com.iota.iri.controllers.MilestoneViewModel;
 import com.iota.iri.hash.SpongeFactory;
 import com.iota.iri.model.Hash;
 import com.iota.iri.model.IntegerIndex;
 import com.iota.iri.service.tipselection.EntryPointSelector;
-import com.iota.iri.storage.Indexable;
-import com.iota.iri.storage.Persistable;
 import com.iota.iri.storage.Tangle;
-import com.iota.iri.utils.Pair;
-import org.junit.Assert;
-import org.junit.Test;
+import com.iota.iri.storage.rocksDB.RocksDBPersistenceProvider;
+import org.junit.*;
+import org.junit.rules.TemporaryFolder;
 import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
 import org.mockito.Mock;
-import org.mockito.Mockito;
 import org.mockito.runners.MockitoJUnitRunner;
 
+import java.util.HashMap;
+
 @RunWith(MockitoJUnitRunner.class)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
 public class EntryPointSelectorImplTest {
+    private static final TemporaryFolder dbFolder = new TemporaryFolder();
+    private static final TemporaryFolder logFolder = new TemporaryFolder();
+    private static Tangle tangle;
 
     @Mock
     private Milestone milestone;
-    @Mock
-    private Tangle tangle;
```

Comment:
The reason tangle was mocked was to increase isolation.
We want `EntryPointSelectorImplTest` to fail because of a bug in `EntryPointSelector`.
We have enough unit tests that use `tangle` as it is that slow the tests down.
Can you return the mock?